samples/subsys/ipc/openamp/CMakeLists.txt

```diff
@@ -17,10 +17,7 @@ enable_language(C ASM)
 # Location of external dependencies:
 set(PLATFORM_DIR "${CMAKE_CURRENT_SOURCE_DIR}/platform")
 
-target_sources(app PRIVATE src/main_master.c
-	       ${PLATFORM_DIR}/platform.c
-	       ${PLATFORM_DIR}/platform_ops.c
-	       ${PLATFORM_DIR}/resource_table.c)
+target_sources(app PRIVATE src/main_master.c)
```

Comment:
I think we should have a "master" folder since we have one for "remote". And a proper README.rst per sample app.

Reply:
I don't have any particular issue with this, so master/main.c? instead of src/main_master.c?  How much do you expect to exist in the master dir beyond the source and a README.rst in both master and remote?

Reply:
I think it makes more clear to have a master/src/main.c and remote/src/main.c, especially for beginners. For master and remote folders we should have a README.rst giving more details what the sample app does alongside with the CMakeLists.txt and prj.conf.

As soon as this and #8527 get merged I will work to get this working on the i.MX7.

Reply:
What do you expect in the remote/README.rst?

Reply:
Introducing to it, how it works. Explaining it is not using remoteproc, etc.